### PR TITLE
The Great T5 Maintaining Adventure

### PR DESCRIPTION
**Pull request by GoldenAlpharex**
## About The Pull Request

All the bad stuff that I found so far and that's actually worth fixing. Mecha and coil will have to be re-fixed next upstream merge because they bot got refactored. Obviously, getting a full monkey out of a dead monkey with a monkey recycler was a bit over-powered, tesla coils having no cooldown between zaps was probably very bad for balance, and mechs generating power when they walk, even in a small amount, just feels dumb.

If you find anything that defies logic anywhere else in the code regarding T5, or just in-game in general, please report it on this PR so I can look into it and fix it.

## Why It's Good For The Game

Why wouldn't fixing T5 be good for the game? It actually stops people from complaining about T5 breaking the game, since it no longer does :)

## Changelog
:cl:
fix: no more infinite monkey recycling loop at T5.
fix: tesla coil now has a 2 seconds cooldown between zaps at T5.
fix: no more generating power when walking in a mech when it has a T5 scanning module.
/:cl:

### CODE REVIEW

**Comment by Coolsurf6** [comment]
Perhaps it is time to yoink this for austation. Watch havok happen.

**Comment by GoldenAlpharex** [comment]
> Perhaps it is time to yoink this for austation. Watch havok happen.

Hahahah, the yoinksurf tide continues.

**Comment by JohnFulpWillard** [comment]
You should also nerf motorized wheelchair to only knock you down for 10 and remove the stamina damage entirely (and maybe nerf some of the damage it causes so it takes a few more rolls to kill)

**Comment by GoldenAlpharex** [comment]
> You should also nerf motorized wheelchair to only knock you down for 10 and remove the stamina damage entirely (and maybe nerf some of the damage it causes so it takes a few more rolls to kill)

Oh, thanks for the reminder! I had completely forgotten about that.

File: code/modules/vehicles/motorized_wheelchair.dm [hunk 1]
```diff
@@ -150,28 +154,20 @@
...
-	if(t5 >= 15 && A.density && isliving(A)) //FULP: If T5 is greater than or equal to 15, run that fucker over and just keep on driving!
-		var/mob/living/D = A
-		var/atom/throw_target = get_edge_target_turf(D, pick(GLOB.cardinals))
-		D.throw_at(throw_target, 2, 3) //FULP: YEET
-		D.Knockdown(85)
-		D.adjustStaminaLoss(40) //FULP: Just like a mulebot, this does no damage on the initial hit- but it does LOADS of damage if you run a prone person over (see Crossed() in carbon.dm for the code for that))
-		D.visible_message("<span class='danger'>[src] rams into [D], sending [D] flying!</span>", "<span class='userdanger'>[src] rams you!</span>")
-		return
-	// FULP: If T5 (T5 identifier) is less than 15, execute as normal
+	///// FULP-ONLY - The Great T5 Maintenance - Removed the snowflake code that basically prevented the person riding a quantum wheelchair from being knocked off. This will nerf its offensive abilities by a lot. - GoldenAlpharex - 09/12/2020
```
Comment (ATH1909) [inline code/modules/vehicles/motorized_wheelchair.dm:157]: you should mention this in the opening post and the changelog

Comment (GoldenAlpharex) [reply code/modules/vehicles/motorized_wheelchair.dm:157]: Good idea. I don't know why I only received the notification about this today.

**Pull request closed without merging**